packages/cli/test/cli/CLITest.ts

```diff
@@ -1,6 +1,6 @@
 import { deepEqual, ok, strictEqual } from 'assert'
 import * as fs from 'fs'
-import { dirname, join } from 'path'
+import { dirname, join, resolve as pathResolve } from 'path'
```

Comment:
I named it `pathResolve` because `resolve` felt so generic that it wouldn't be obvious to other devs that it comes from `path`.